### ISSUE TITLE
intl ^0.17.0 error

### ISSUE THREAD

**Issue by hm-tamim** [GiancarloCode/form_bloc#334]
Getting this error:

```
Because flutter_form_bloc 0.30.1 depends on intl ^0.17.0 and no versions of flutter_form_bloc match >0.30.1 <0.31.0, flutter_form_bloc ^0.30.1 requires intl ^0.17.0.
So, because **** depends on both flutter_form_bloc ^0.30.1 and intl ^0.18.1, version solving failed.
```

**Comment by mghase** [comment]
check my solution here
[](https://stackoverflow.com/a/76361785/6664250)

**Comment by GiancarloCode** [comment]
Solved in v0.31.0 🎉
